### PR TITLE
recent plugin- and build-related updates

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

Branch to include work related to the UN women online servers

cc @jredrejo @rtibbles 

### Reviewer guidance

…

### References

…

----

### Contributor Checklist

PR process:

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: docker/env.list [hunk 1]
```diff
@@ -29,3 +29,5 @@ KOLIBRI_RUN_MODE="docker"
 # DOCKERMNT_PEX_PATH=
 # Specify path to a .pex file in the /docker/mnt volume, e.g.
 # DOCKERMNT_PEX_PATH=/docker/mnt/kolibri-0.11.0a1.dev_git-37-gb15b923.pex
+
+BACKGROUND_IMAGE
```
Comment (indirectlylit) [inline docker/env.list:33]: maybe we should rename this to something namespaced and build-specific like `KOLIBRI_BUILD_BG_IMAGE` .. somehow distinguish it from kolibri options?

**Comment by benjaoming** [comment]
~This is targeted for `0.12.x`?~

Okay, I can see that we don't have `0.12.y` anymore, which confused me. Before and after it got deleted.

Now my question is: Weren't all these changes intended for 0.13 @rtibbles ?

**Comment by rtibbles** [comment]
They were originally intended for 0.13 - but then became important for the UN Women work.

**Comment by radinamatic** [comment]
Not sure which I like more:
`monkey patching hack` itself, or the `Fixing redirection monkey patching redirect_to` 
🥇 🏆

**Comment by indirectlylit** [comment]
merging so we can tag an alpha

**Pull request merged**